controller/api/models.py

```diff
@@ -643,7 +643,8 @@ def create(self):
         kwargs = {'memory': self.release.config.memory,
                   'cpu': self.release.config.cpu,
                   'tags': self.release.config.tags,
-                  'values': self.release.config.values}
+                  'values': self.release.config.values,
+                  'aname': self.app.id}
```

Comment:
https://github.com/Hearst-Hatchery/deis/blob/ccc32b9b9710c9e79bbdf866c98550daa7d7ea95/controller/api/models.py#L336
https://github.com/Hearst-Hatchery/deis/blob/ccc32b9b9710c9e79bbdf866c98550daa7d7ea95/controller/api/models.py#L487

It appears to not actually be used anywhere after passed through, though. Probably vestigial and I imagine can be safely renamed.